lib/constants/locales.js

```diff
@@ -11,6 +11,7 @@
 // Please keep English at the top, and sort other entries by `name`.
 export default {
   en: { name: 'English', nativeName: 'English', completion: '100%', flag: '🇺🇸' },
+  'pt-Br': { name: 'Brazilian Portuguese', nativeName: 'Português brasileiro', completion: '100%', flag: '🇧🇷' },
```

Comment:
shouldn't it be the same as the crowdin file name?
```suggestion
  'pt-BR': { name: 'Brazilian Portuguese', nativeName: 'Português brasileiro', completion: '100%', flag: '🇧🇷' },
```